packages/unmock-core/src/generator.ts

```diff
@@ -51,10 +55,33 @@ export function responseCreatorFactory({
   );
   const serviceStore = new ServiceStore(services);
   const stateStore = stateFacadeFactory(serviceStore);
+  const toIndentedYaml = (
+    input: any, // Adds one more indentation to each line
+  ) =>
+    yaml
+      .dump(input.body ? { ...input, body: JSON.parse(input.body) } : input)
+      .split("\n")
+      .map(
+        (line: string) =>
+          `\t${line.replace("!<tag:yaml.org,2002:js/undefined> ''", "")}`,
```

Comment:
Removes `js/undefined` tag; maybe we want to keep it? Or maybe we want to use JSON format? I felt yaml is more readable as a log.